src/uct/tcp/tcp_iface.c

```diff
@@ -86,6 +87,10 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
    "let the operating system select the port number.",
    ucs_offsetof(uct_tcp_iface_config_t, port_range), UCS_CONFIG_TYPE_RANGE_SPEC},
 
+   {"MAX_BW", "2200MBs",
```

Comment:
maybe introduce support for `inf` value for `UCS_CONFIG_TYPE_BW`?
@yosefe, wdyt?

Reply:
Aren't we going to use `inf` instead?
ping @yosefe

Reply:
auto would produce the same effect as infinite